app/screens/thread/index.tsx

```diff
@@ -3,17 +3,27 @@
 
 import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
 import withObservables from '@nozbe/with-observables';
+import {of as of$} from 'rxjs';
+import {distinctUntilChanged, switchMap} from 'rxjs/operators';
 
+import {observeCurrentCall} from '@calls/state';
+import {withServerUrl} from '@context/server';
 import {observePost} from '@queries/servers/post';
 
 import Thread from './thread';
 
 import type {WithDatabaseArgs} from '@typings/database/database';
 
 const enhanced = withObservables(['rootId'], ({database, rootId}: WithDatabaseArgs & {rootId: string}) => {
+    const isInACall = observeCurrentCall().pipe(
+        switchMap((call) => of$(Boolean(call))),
+        distinctUntilChanged(),
+    );
+
     return {
         rootPost: observePost(database, rootId),
+        isInACall,
     };
 });
 
-export default withDatabase(enhanced(Thread));
+export default withDatabase(withServerUrl(enhanced(Thread)));
```

Comment:
Where is the serverUrl being used?

Reply:
Sorry, must have been refactored out and I forgot to remove it. :) Fixed.